from rest_framework.exceptions import APIException

class UserDirectoryAlreadyExists(APIException):
    pass